test/test_functional_tensor.py

```diff
@@ -692,6 +433,250 @@ def test_scale_channel(self):
         assert_equal(scaled_cpu, scaled_cuda.to('cpu'))
 
 
+class TestAffine:
+
+    @pytest.mark.parametrize('device', cpu_and_gpu())
+    @pytest.mark.parametrize('height, width', [(26, 26), (32, 26)])
+    @pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
+    def test_identity_map(self, device, height, width, dt):
+        # Tests on square and rectangular images
+        scripted_affine = torch.jit.script(F.affine)
```

Comment:
to avoid scripting a function in each function, maybe we should declare this as a class variable as well

Reply:
yes that's the best way.